internal/execute/outputs.go

```diff
@@ -260,13 +261,165 @@ func generateOptionOutput(
+				option.Elements(), option,
+			),
+		)
+	}
 
-	// !!!
+	var terminalWidth int
+	// const terminalWidth = sys.getWidthOfTerminal?.() ?? 0;
+
+	// Note: child_process might return `terminalWidth` as undefined.
+	if terminalWidth >= 80 {
+		// !!!     let description = "";
+		// !!!     if (option.description) {
+		// !!!         description = getDiagnosticText(option.description);
+		// !!!     }
+		// !!!     text.push(...getPrettyOutput(name, description, rightAlignOfLeft, leftAlignOfRight, terminalWidth, /*colorLeft*/ true), sys.newLine);
+		// !!!     if (showAdditionalInfoOutput(valueCandidates, option)) {
+		// !!!         if (valueCandidates) {
+		// !!!             text.push(...getPrettyOutput(valueCandidates.valueType, valueCandidates.possibleValues, rightAlignOfLeft, leftAlignOfRight, terminalWidth, /*colorLeft*/ false), sys.newLine);
+		// !!!         }
+		// !!!         if (defaultValueDescription) {
+		// !!!             text.push(...getPrettyOutput(getDiagnosticText(Diagnostics.default_Colon), defaultValueDescription, rightAlignOfLeft, leftAlignOfRight, terminalWidth, /*colorLeft*/ false), sys.newLine);
+		// !!!         }
+		// !!!     }
+		// !!!     text.push(sys.newLine);
+	} else {
+		text = append(text /* !!! colors.blue(name) */, name, sys.NewLine())
+		if option.Description != nil {
+			text = append(text, option.Description.Format())
+		}
+		text = append(text, sys.NewLine())
+		if showAdditionalInfoOutput(valueCandidates, option) {
+			if valueCandidates != nil {
+				text = append(text, valueCandidates.valueType, " ", valueCandidates.possibleValues)
+			}
+			if defaultValueDescription != "" {
+				if valueCandidates != nil {
+					text = append(text, sys.NewLine())
+				}
+				text = append(text, diagnostics.X_default_Colon.Format(), " ", defaultValueDescription)
+			}
+
+			text = append(text, sys.NewLine())
+		}
+		text = append(text, sys.NewLine())
+	}
 
 	return text
 }
 
+func formatDefaultValue(defaultValue any, option *tsoptions.CommandLineOption) string {
+	if defaultValue == core.TSUnknown {
+		return "undefined"
+	}
+
+	if option.Kind == tsoptions.CommandLineOptionTypeEnum {
+		// e.g. ScriptTarget.ES2015 -> "es6/es2015"
+		var names []string
+		for name, value := range option.EnumMap().Entries() {
+			if value == defaultValue {
+				names = append(names, name)
+			}
+		}
+		return strings.Join(names, "/")
+	}
+	return fmt.Sprintf("%v", defaultValue)
+}
+
+type valueCandidate struct {
+	// "one or more" or "any of"
+	valueType      string
+	possibleValues string
+}
+
+func showAdditionalInfoOutput(valueCandidates *valueCandidate, option *tsoptions.CommandLineOption) bool {
+	if option.Category == diagnostics.Command_line_Options {
+		return false
+	}
+	if valueCandidates.possibleValues == "string" && (option.DefaultValueDescription == nil ||
+		option.DefaultValueDescription == "false" ||
+		option.DefaultValueDescription == "n/a") {
+		return false
+	}
```

Comment:
```suggestion
    if valueCandidates != nil && valueCandidates.possibleValues == "string" && (
        option.defaultValueDescription == nil ||
    	option.DefaultValueDescription == "false" ||
		option.DefaultValueDescription == "n/a"
    ) {
		return false
	}
```

Reply:
I think we can not put a `newline` between `"n/a"` and `)` which results in syntax error in `go`. And the extra `newline` after `(` will be removed by `go fmt`. :thinking:

Reply:
Thats fine! I'm never sure of how `go fmt` ends up formatting statements 😅. Can you add the `valueCandidates != nil` statement?